src/App.css

```diff
@@ -1,26 +1,49 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto+Slab&display=swap');
+
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
 }
 
+body {
+  background-color: #fafafa;
+}
+
+/*  Navbar */
 .navBar {
-  height: 4rem;
   display: flex;
-  align-items: center;
-  padding: 0 2rem;
+  flex-direction: row;
   justify-content: space-between;
+  position: relative;
+  align-items: center;
+  font-family: 'Montserrat', sans-serif;
+  padding: 1% 5%;
+  background-color: #fff;
+  width: 100%;
+}
```

Comment:
- [ ] As suggested by the previous reviewer,

There is one important aspect that needs attention: the alignment of the navbar and its content. To ensure consistency with the Zeplin design, please make sure to align the navbar elements as described. Here are some guidelines to follow:

Positioning: Ensure that the navbar is positioned correctly according to the Zeplin design. Pay attention to any specified margins or paddings.

Element Alignment: Make sure that the navbar elements, such as the logo, navigation links, and any other components, are aligned as described in the Zeplin design. This includes both vertical and horizontal alignment.

Responsive Design: Consider the responsiveness of the navbar and ensure that it behaves as expected on different screen sizes. This might involve adjusting the alignment and layout for smaller screens.

By aligning the navbar and its content as described in the Zeplin design, you will achieve a consistent and visually appealing layout. If you have any questions or need further clarification, please feel free to ask. Keep up the great work! 💪